experimental/packages/opentelemetry-instrumentation-http/test/integrations/https-enable.test.ts

```diff
@@ -16,11 +16,13 @@
 
 import { SpanKind, Span, context, propagation } from '@opentelemetry/api';
 import {
-  HTTPFLAVORVALUES_HTTP_1_1,
-  NETTRANSPORTVALUES_IP_TCP,
-  SEMATTRS_HTTP_FLAVOR,
-  SEMATTRS_NET_TRANSPORT,
+  ATTR_NETWORK_PROTOCOL_NAME,
+  ATTR_NETWORK_TRANSPORT,
 } from '@opentelemetry/semantic-conventions';
+import {
+  NET_TRANSPORT_VALUE_IP_TCP,
+  HTTP_FLAVOR_VALUE_HTTP_1_1,
+} from '@opentelemetry/semantic-conventions/incubating'
```

Comment:
See comments above about not using these exports from the `incubating` entry-point.